### DISCUSSION TITLE
Separate Border styles

### DISCUSSION THREAD

**Discussion by Poimen** [tailwindlabs/tailwindcss#9074] [Help] [answered]
We have a UI requirement that on an error, the `div` box containing the error condition should have a dashed border top, left and right, with a solid bottom border style.

I couldn't find a way of doing this with the current utilities or the JIT implementation - i.e. customise the border style per side, something like `border-b-solid`.

Outside of extending the config/css to contain a special border class, is there a way of doing this with the utilities?

**Comment by wongjn** [comment, marked as answer]
Not sure if this constitutes "extending the config" but you could write a plugin that adds per-side border style utilities? Otherwise, you could use the arbitrary utility class, `[border-bottom-style:solid]` like in [this Tailwind Play](https://play.tailwindcss.com/AiJ3lIWgbL).

**Reply by Poimen** [reply]
`[border-bottom-style:solid]` is awesome - I was trying to get it to work like `border-b-[style: solid]` 🤦‍♂️ 

Thanks 👍
